workflows/cloud-client/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+  Copyright 2021, Google LLC
```

Comment:
the formatting here looks different? Can you confirm? https://github.com/GoogleCloudPlatform/java-docs-samples#source-code-headers

Reply:
Updated. Thanks Kristen!